projects/packages/my-jetpack/_inc/hooks/use-notification-watcher/use-site-connection-notice.ts

```diff
@@ -39,23 +79,47 @@ const useSiteConnectionNotice = ( redBubbleAlerts: RedBubbleAlerts ) => {
 						'jetpack-my-jetpack'
 				  );
 
+		const needsSiteConnectionMessage = __(
+			'Some products need a connection to WordPress.com to be able to work.',
+			'jetpack-my-jetpack'
+		);
```

Comment:
With the site connection message - we need a way to include our TOS copy on the banner to indicate that clicking on the connection button is agreeing to our TOS.

We discussed handling that in another small PR - let's merge that together with this one before it goes into trunk.